oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/meter/MeterEntity.java

```diff
@@ -35,6 +36,8 @@
 @Getter
 @Builder(toBuilder = true)
 public class MeterEntity {
+    private static NamingControl NAMING_CONTROL;
```

Comment:
Where is it used?

Reply:
I forgot to submit the second change. Just did.